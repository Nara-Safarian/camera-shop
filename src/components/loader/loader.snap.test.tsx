import { render } from '@testing-library/react';
import Loader from './loader';

test('Loader snapshot should be rendered correctly', () => {
  const {container} = render(
    <Loader />
  );
  expect(container).toMatchSnapshot();
});
