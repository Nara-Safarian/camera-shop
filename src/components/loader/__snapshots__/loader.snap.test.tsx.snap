// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader snapshot should be rendered correctly 1`] = `
<div>
  <div
    class="three-body-wrapper"
  >
    <div
      class="three-body"
    >
      <div
        class="three-body__dot"
      />
      <div
        class="three-body__dot"
      />
      <div
        class="three-body__dot"
      />
    </div>
  </div>
</div>
`;
